config/settings/common_settings.py

```diff
@@ -164,10 +164,10 @@ def pipe_delim(pipe_string):
 
 MEDIA_URL = "/media/"
 
-# Update Authentication classes, removed BasicAuthentication
 # Defaults: https://www.django-rest-framework.org/api-guide/settings/
 REST_FRAMEWORK = {
     'DEFAULT_AUTHENTICATION_CLASSES': [
+        'rest_framework.authentication.BasicAuthentication',
```

Comment:
`BasicAuthentication` has many [security vulnerabilities](https://www.whiteoaksecurity.com/blog/simple-security-fails-part-5-basic-authentication/), we should not be using it and I don't see why it is needed for to fix this issue

Reply:
Replaced with `TokenAuthentication`.

I was unaware of the security vulnerabilities of `BasicAuthentication` as they weren't mentioned any where in the [docs](https://www.django-rest-framework.org/api-guide/authentication/#basicauthentication).

>  I don't see why it is needed for to fix this issue

`BasicAuthentication` did provide the appropriate `WWW-Authenticate` header which was necessary for returning `HTTP 401 Unauthorized` which was lacking in `SessionAuthentication` therefore I decided to go with it.